connectivity/components.cmake

```diff
@@ -0,0 +1,16 @@
+# Copyright (c) 2020 ARM Limited. All rights reserved.
+# SPDX-License-Identifier: Apache-2.0
+
+list(APPEND connectivity-components
+    mbed-os-cellular
+    mbed-os-ble
+    mbed-os-lorawan
+    mbed-os-mbedtls
+    mbed-os-nanostack
+    mbed-os-netsocket
+    mbed-os-nfc
+)
+
+# only BLE folder is named differently
```

Comment:
I'll fix this comment and also paths below. It is a table of what is up there and paths. So our scripts will always do NAME + _PATH to find the path to a component (as name could be different than location, etc)

Reply:
fixed